iceoryx-sys/src/lib.rs

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: © Contributors to the iceoryx-rs project
+// SPDX-FileContributor: Mathias Kraus
+
+#![recursion_limit = "256"]
+
+#[macro_use]
+extern crate cpp;
+
+pub mod introspection;
+
+mod publisher;
+pub use publisher::Publisher;
+
+mod publisher_options;
+pub use publisher_options::PublisherOptions;
+
+mod queue_policy;
+pub use queue_policy::ConsumerTooSlowPolicy;
+pub use queue_policy::QueueFullPolicy;
+
+mod runtime;
+pub use runtime::Runtime;
+
+mod subscriber;
+pub use subscriber::ConditionVariable;
+pub use subscriber::SubscribeState;
+pub use subscriber::Subscriber;
+pub use subscriber::SubscriberArc;
```

Comment:
What does Arc mean in this context?

Reply:
It means `Atomically Reference Counted` and indicates that the lifetime of the subscriber is managed by a thread-safe reference-counting pointer